Lib/lib2to3/tests/__init__.py

```diff
@@ -1,8 +1,11 @@
 # Author: Collin Winter
 
 import os
+import warnings
 
 from test.support import load_package_tests
 
 def load_tests(*args):
-    return load_package_tests(os.path.dirname(__file__), *args)
+    with warnings.catch_warnings():
+        warnings.filterwarnings('ignore', category=DeprecationWarning, module='lib2to3')
```

Comment:
```suggestion
        warnings.filterwarnings('ignore', category=DeprecationWarning, message='lib2to3')
```
I tried locally and `module=` doesn't work, because the module where the warning is raised is actually `unittest.loader`. Let's match on the message instead.